web/src/views/Repos.vue

```diff
@@ -8,30 +8,42 @@
       <Button :to="{ name: 'repo-add' }" start-icon="plus" :text="$t('repo.add')" />
     </template>
 
-    <div class="space-y-4">
-      <ListItem v-for="repo in searchedRepos" :key="repo.id" :to="{ name: 'repo', params: { repoId: repo.id } }">
-        <span class="text-wp-text-100">{{ `${repo.owner} / ${repo.name}` }}</span>
-      </ListItem>
+    <div class="flex flex-col gap-12">
+      <div class="gap-4 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2">
+        <RepoItems v-for="repo in repoListAccess" :key="repo.id" :repo="repo" />
+      </div>
+
+      <div class="flex flex-col gap-4">
+        <RepoItems v-for="repo in repoListActivity" :key="repo.id" :repo="repo" />
+      </div>
     </div>
   </Scaffold>
 </template>
 
 <script lang="ts" setup>
-import { computed, onMounted, ref } from 'vue';
+import { computed, ref } from 'vue';
+import { useRouter } from 'vue-router';
 
 import Button from '~/components/atomic/Button.vue';
-import ListItem from '~/components/atomic/ListItem.vue';
 import Scaffold from '~/components/layout/scaffold/Scaffold.vue';
+import RepoItems from '~/components/repo/RepoItems.vue';
+import useRepos from '~/compositions/useRepos';
 import { useRepoSearch } from '~/compositions/useRepoSearch';
 import { useRepoStore } from '~/store/repos';
 
+const router = useRouter();
+
 const repoStore = useRepoStore();
 const repos = computed(() => Object.values(repoStore.ownedRepos));
 const search = ref('');
 
 const { searchedRepos } = useRepoSearch(repos, search);
+const { sortReposByLastAccess, sortReposByLastActivity } = useRepos();
+
+const repoListAccess = computed(() => sortReposByLastAccess(repos.value || []));
+const repoListActivity = computed(() => sortReposByLastActivity(searchedRepos.value || []));
 
-onMounted(async () => {
+router.beforeEach(async () => {
```

Comment:
This would load the list again each time you navigate to a route (not only this one).

Reply:
Yeah, this was more or less a hack anyway where I wanted to ask for help :see_no_evil: `onMount` behaves weird when moving back and forward in the browser history e.g. the last duration is missing  (see video). Do you have an idea why?

https://github.com/user-attachments/assets/8a8a4b8e-d895-46c4-b693-460fcd58bee2

Reply:
I think when opening the repo it loads the specific repo again and overwrites `last_pipeline` with the number and therefore you loose the `last_pipeline` data (see my other comment).

Reply:
Ok even if I dont understand where `last_pipeline` (the number) is coming from I have added a new property `last_pipeline_item` as you suggested and this seems to fix the problem. Thanks!

Reply:
@anbraten looks like its still not fixed -.- I still see missing information while using the browser navigation. Any other ideas?

Reply:
Did some adjustments to setting the repos in store as it was overriding the entries again when opening a repo page.